lighthouse-core/report/html/renderer/text-encoding.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* global self btoa atob window CompressionStream Response */
+
+const btoaIso = typeof btoa !== 'undefined' ?
```

Comment:
Iso?

Reply:
isomorphic

Reply:
prefer `btoa_`?

Reply:
ah gotcha, yeah `btoa_` was what I had in my head, or `btoaIsomorphic` :D

Reply:
i think there's fair consensus that 'atob' is one of the worst named APIs of the web…  so i don't love calling our stuff that.

should either go full out `asciiToBinary` if you wanna honor that API or otherwise something like `textToBase64String`. I'm fine leaving off the isomophoric suffix, but could add a comment above to explain why we do it so funny

Reply:
> i think there's fair consensus that 'atob' is one of the worst named APIs of the web… so i don't love calling our stuff that.

I totally agree, and I fully support never naming our APIs `atob` or `btoa`.

*However*, internal to this I think there's value in reflecting what underlying methods are being used to perform this so I'd strongly prefer ` asciiToBinary` to a complete rename. We already did the complete useful rename for the API surface of this module :)

Reply:
sg